apps/wallet/src/ui/app/components/ledger/useDeriveLedgerAccounts.ts

```diff
@@ -73,8 +80,10 @@ async function deriveAccountsFromLedger(
         );
         const publicKey = new Ed25519PublicKey(publicKeyResult.publicKey);
         ledgerAccounts.push({
+            type: AccountType.LEDGER,
```

Comment:
Do we need to store which ledger this came from (for multi ledger support)? Not sure that's possible though.

Reply:
IIRC I think we do have some device information, I can work on multi-ledger support in a fast-follow!